src/components/NoteList.vue

```diff
@@ -1,26 +1,83 @@
 <template>
   <section class="screen-box">
```

Comment:
I have a question about HTML coding.
In the parent component, is the following image acceptable?
```
<section class = "screen-box">
   <Header title = "[page title]" url = "[return url]"> </Header>
   <main class = "screen-box__main">
       content
   </main>
</section>
```

Reply:
Don't should have spaces when declare attributes of HTML tags